### DISCUSSION TITLE
react-map-gl v7 not working in Safari < 13.1 due to optional chaining

### DISCUSSION THREAD

**Discussion by timsluis** [uber/react-map-gl#1791] [Q&A] [answered]
The ESM output is transpiled to `es2020`, which includes optional chaining (?.). Safari versions below 13.1 don't support optional chaining resulting in the following error: `​SyntaxError: Unexpected token '.'`

Should I change something in my configuration? Or should the `target` attribute be set to `es2019` in tsconfig.esm.json, so the transpiled output will support a wider range of browsers?

**Comment by Pessimistress** [comment, marked as answer]
You need to adjust your bundler's transpile settings to support older versions of browsers. The library is only lightly transpiled so that you have control of what environment you want to target.

If you do not have access to the build configuration, there is a dist/es5 directory that is fully transpiled.
